src/engine/controls/loopingcontrol.h

```diff
@@ -93,7 +93,7 @@ class LoopingControl : public EngineControl {
     void slotLoopHalve(double pressed);
 
   private:
-    enum LoopSeekMode {
+    enum class LoopSeekMode {
         CHANGED, // force the playposition to be inside the loop after adjusting it.
```

Comment:
We conventionally use CamelCase ("Changed", "MovedOut", "None") for enum class values.